Odata-docs/webapi-8/fundamentals/overview.md

```diff
@@ -0,0 +1,578 @@
+
+```json
+POST http://localhost:5000/$batch
+Content-Type: application/json
+
+{
+    "requests": [
+        {
+            "method": "GET",
+            "url": "http://localhost:5000/Products/1",
+            "id": "r1"
+        },
+        {
+            "method": "GET",
+            "url": "http://localhost:5000/People?$select=FirstName&$top=2",
+            "id": "r2"
+        },
+        {
+            "method": "PATCH",
+            "url": "http://localhost:5000/Products/2",
+            "id": "r3",
+            "headers": {
+                "content-type": "application/json"
+            },
+            "body": {
+                "Category": "Electronics"
+            }
+        }
+    ]
+}
+
+```
+
+**Response**
+
+The response body contains an array of responses, on for each request. Each response includes the status, headers, body and other metadata that allows the client to process the response.
+
+```json
+{
+    "responses": [
+        {
+            "id": "r1",
+            "status": 200,
+            "headers": {
+                "content-type": "application/json; odata.metadata=minimal; odata.streaming=true; charset=utf-8",
+                "odata-version": "4.0"
+            },
+            "body": {
+                "@odata.context": "http://localhost:5000/$metadata#Products/$entity",
+                "Id": 1,
+                "Category": "Goods",
+                "Color": "Red",
+                "CreatedDate": "2001-04-15T16:24:08-08:00",
+                "UpdatedDate": "2011-02-15T16:24:08-08:00"
+            }
+        },
+        {
+            "id": "r2",
+            "status": 200,
+            "headers": {
+                "content-type": "application/json; odata.metadata=minimal; odata.streaming=true; charset=utf-8",
+                "odata-version": "4.0"
+            },
+            "body": {
+                "@odata.context": "http://localhost:5000/$metadata#People(FirstName)",
+                "value": [
+                    {
+                        "FirstName": "Fiction"
+                    },
+                    {
+                        "FirstName": "Goods"
+                    }
+                ]
+            }
+        },
+        {
+            "id": "r3",
+            "status": 200,
+            "headers": {
+                "content-type": "application/json; odata.metadata=minimal; odata.streaming=true; charset=utf-8",
+                "odata-version": "4.0"
+            },
+            "body": {
+                "@odata.context": "http://localhost:5000/$metadata#Products/$entity",
+                "Id": 2,
+                "Category": "Electronics",
+                "Color": "Blue",
+                "CreatedDate": "2021-12-27T09:12:08-08:00",
+                "UpdatedDate": null
+            }
+        }
+    ]
+}
+```
+
+This batch request contained the requests:
```

Comment:
```suggestion
This batch request contained the following requests:
```

Reply:
I opted to remove this list altogether since it's redundant.